client/src/routes/+layout.svelte

```diff
@@ -13,15 +13,158 @@
   aria-describedby="rules-dialog-content"
 >
   <Header>
-    <Title id="rules-dialog-title">Rules!</Title>
+    <Title id="rules-dialog-title">How to Play</Title>
     <IconButton action="close" class="material-icons">close</IconButton>
   </Header>
   <Content id="rules-dialog-content">
-    <!-- TODO: Rules Content Here -->
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
-    irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-    cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    <h3 class="mdc-typography--headline5">Setting</h3>
+    <p class="mdc-typography--body1">
+      You are a very lonely cat, looking for another furry friend so you decide to try out online dating. In comes a
+      brand new, top-of-the-line dating app, Cat Me if You Can! You’ve heard great things about the world of online
+      dating and other cats instantly finding matches from other cats with owners from all across the world. Everything
+      is great except for those slimy catfish that try to swim their way through this cat-only dating app and deceive
+      all the innocent other cats. Your job is to remove these pesky catfish from the app and keep the app fish-free.<br
+      />
+    </p>
+
+    <h3 class="mdc-typography--headline5">Recommendation</h3>
+    <ul class="mdc-typography--body1">
+      <li>Play the game while being able to speak with other players.</li>
+      <li>Do not talk to other players during the game until the voting phase has arrived.</li>
+    </ul>
+
+    <h3 class="mdc-typography--headline5">Phases</h3>
+    <ul class="mdc-typography--body1">
+      <li>Lobby</li>
+      <li>Recieve A Role</li>
+      <li>Answer Prompt</li>
+      <li>Chatting</li>
+      <li>Voting</li>
+      <li>End</li>
+    </ul>
```

Comment:
I feel like this isn't needed because we have the headings. In the future maybe we will condence the rules because currently it is kinda long